src/Tutor.Core/UseCases/Management/Stakeholders/CourseOwnershipService.cs

```diff
@@ -29,7 +31,13 @@ public Result<Course> UpdateOwnedCourse(Course course, int instructorId)
         var isOwner = _ownedCourseRepository.IsCourseOwner(course.Id, instructorId);
         if (!isOwner) return Result.Fail(FailureCode.Forbidden);
 
-        return _courseRepository.Update(course);
+        _courseRepository.Update(course);
+        // DEMO: context caching (with and without save called)
```

Comment:
I imagine this is a reminder for our meeting, but pointing it out to avoid merging this code.